src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -23,7 +23,7 @@ export class DocService {
     });
   }
 
-  urlGenerator(release: string, section: string): string {
+  urlGenerator(section: string, release?: string): string {
```

Comment:
Its probably better to have a default value, to prevent bad links.
```suggestion
  urlGenerator(section: string, release = 'master'): string {
```

Reply:
Yes.

Reply:
Done.